views/building-hours/all-building-hours.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import {
+  Text,
+} from 'react-native'
+
+import moment from 'moment'
+
+export function allBuildingHours(info, style) {
+  let dayTimes = []
+  let hoursString = ''
+  for (let i = 0; i < 7; i++) {
+    let day = moment().add(i, 'days').format('dddd')
+    let d = moment().add(i, 'days').format('ddd')
```

Comment:
Two notes here: one, could you create one moment and query it twice, instead of creating two moments? I have an irrational fear of the two moments getting created across a day divide so they don't match… 

The other note is, could you use moment-timezone set to … I think we use America/Winnipeg? Otherwise is used the device's timezone, which is fine but the rest of the building hours stuff uses the explicit America/Winnipeg zone. I think it'll be really confused when I try it in Japan if half of it's in JST and half is in CST…